src/internal_modules/roc_audio/latency_monitor.cpp

```diff
@@ -250,13 +253,16 @@ void LatencyMonitor::report_latency_(packet::timestamp_diff_t latency) {
     }
 
     roc_log(LogDebug, "latency monitor: latency=%ld(%.3fms) target=%lu(%.3fms)",
-            (long)latency,
-            (double)input_sample_spec_.rtp_timestamp_2_ns(latency) / core::Millisecond,
+            (long)latency, (double)latency_ / core::Millisecond,
             (unsigned long)target_latency_,
             (double)input_sample_spec_.rtp_timestamp_2_ns(
                 (packet::timestamp_diff_t)target_latency_)
                 / core::Millisecond);
 }
 
+core::nanoseconds_t LatencyMonitor::latency() const {
+    return latency_;
+}
```

Comment:
Giving it second thought... This local latency fields is very arguable. It will oscillate a lot because of network jitter, so this is probably not something that the user would expect from "latency of pipeline".

I suggest to remove local latency for now, and add only end to end latency in this PR.

Reply:
May be we could get smoothed local latency from `FrequencyEstimator`, but it needs some experiments.